### PR TITLE
Add slugify on slug field in Channel mutations

### PR DESCRIPTION
**Pull request by d-wysocki**
I want to merge this change because...I want to use slugify on slug field in ChannelCreate and ChannelUpdate mutation

# Impact

* [ ] New migrations
* [ ] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [ ] The changes are tested
* [ ] GraphQL schema and type definitions are up to date
* [ ] Changes are mentioned in the changelog

### CODE REVIEW

File: saleor/graphql/channel/mutations.py [hunk 1]
```diff
@@ -65,6 +75,15 @@ class Meta:
         error_type_class = ChannelError
         error_type_field = "channel_errors"
 
+    @classmethod
+    def clean_input(cls, info, instance, data, input_cls=None):
+        clean_input = super().clean_input(info, instance, data)
```
Comment (fowczarek) [inline saleor/graphql/channel/mutations.py:80]:
```suggestion
        cleaned_input = super().clean_input(info, instance, data)
```

File: saleor/graphql/channel/mutations.py [hunk 2]
```diff
@@ -45,6 +46,15 @@ class Meta:
     def get_type_for_model(cls):
         return Channel
 
+    @classmethod
+    def clean_input(cls, info, instance, data, input_cls=None):
+        clean_input = super().clean_input(info, instance, data)
```
Comment (fowczarek) [inline saleor/graphql/channel/mutations.py:51]:
```suggestion
        cleaned_input = super().clean_input(info, instance, data)
```

File: saleor/graphql/channel/tests/test_channel_create_mutations.py [hunk 3]
```diff
@@ -125,9 +127,8 @@ def test_channel_create_mutation_with_invalid_slug(
     content = get_graphql_content(response)
 
     # then
-    error = content["data"]["channelCreate"]["channelErrors"][0]
-    assert error["field"] == "slug"
-    assert error["code"] == ChannelErrorCode.INVALID.name
+    channel_data = content["data"]["channelCreate"]["channel"]
+    assert channel_data["slug"] == slugify(slug)
```
Comment (fowczarek) [inline saleor/graphql/channel/tests/test_channel_create_mutations.py:131]: We should rename this test.

File: saleor/graphql/channel/tests/test_channel_update_mutations.py [hunk 4]
```diff
@@ -126,9 +127,8 @@ def test_channel_update_mutation_with_invalid_slug(
     content = get_graphql_content(response)
 
     # then
-    error = content["data"]["channelUpdate"]["channelErrors"][0]
-    assert error["field"] == "slug"
-    assert error["code"] == ChannelErrorCode.INVALID.name
+    channel_data = content["data"]["channelUpdate"]["channel"]
+    assert channel_data["slug"] == slugify(slug)
```
Comment (fowczarek) [inline saleor/graphql/channel/tests/test_channel_update_mutations.py:131]: We should rename this test.

Comment (d-wysocki) [reply saleor/graphql/channel/tests/test_channel_update_mutations.py:131]: The name of test is `test_channel_update_mutation_with_invalid_slug` and in this case im testing update channel with invalid slug. IMO this name is good, any sugestions how i should name it?

Comment (fowczarek) [reply saleor/graphql/channel/tests/test_channel_update_mutations.py:131]: Maybe `test_channel_update_mutation_slugify_slug_field` ?

**Review by maarcingebala — APPROVED** [review]

**Review by IKarbowiak — APPROVED** [review]

**Review by fowczarek — APPROVED** [review]

**Pull request merged**